src/commands/cluster-slot-stats.json

```
@@ -0,0 +1,83 @@
+{
+    "SLOT-STATS": {
+        "summary": "Return array of slot usage statistics for slots assigned to the current node",
+        "complexity": "O(N) where N is the total number of slots based on arguments. O(N log N) with ORDERBY subcommand.",
+        "group": "cluster",
+        "since": "8.0.0",
+        "arity": -2,
```

```suggestion
        "arity": -4,
```
Since we dropped the two argument form, the minimum seems to be 4 now. This is kind of important since you'll get a weird error if you drop an argument.